### PR TITLE
Add shortcut support

### LINKED ISSUE #5: Add shortcuts to generate furiganas
This can be handy, add keyboard shortcuts to generate furiganas !

### PR DESCRIPTION
**Pull request by shouya**
This PR adds configurable shortcut support. I think it should fix #5.

Thank you for the nice addon!

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {
-   "useRubyTags": false,
-   "ignoreNumbers": true
- }
+    "useRubyTags": false,
+    "ignoreNumbers": true,
+    "keyboardShortcut": {
+        "add_furigana": "ctrl+alt+f",
```
Comment (shouya) [inline config.json:5]:
```suggestion
        "add_furigana": "Alt+F",
```

File: config.json [hunk 2]
```diff
@@ -1,4 +1,8 @@
 {
-   "useRubyTags": false,
-   "ignoreNumbers": true
- }
+    "useRubyTags": false,
+    "ignoreNumbers": true,
+    "keyboardShortcut": {
+        "add_furigana": "Alt+F",
+        "del_furigana": "ctrl+alt+F"
```
Comment (shouya) [inline config.json:6]:
```suggestion
        "del_furigana": "Ctrl+Alt+F"
```

**Comment by obynio** [comment]
Whoa cool work, let me get back to you once I reviewed this !

**Comment by obynio** [comment]
@shouya thank you for the nice work !

droppedthis

**Pull request merged**